### PR TITLE
UI: Fonts

### PR DESCRIPTION
**Pull request by kpyszkowski**
Ref: #2 

This pull request replaces fonts packages to follow designs guidelines.
Currently used fonts are [DM Sans](https://fonts.google.com/specimen/DM+Sans) and [Bricolage Grotesque](https://fonts.google.com/specimen/Bricolage+Grotesque).

Initially font faces presented in Figma designs are [Aeonik Pro](https://aeonik.co.uk/font/) and [Neue Machina](https://pangrampangram.com/products/neue-machina). Those have been (temporarily) replaced by free-licensed alternatives.

Note: I've made a typo in commit messages - the font property that uses the _Bricolage Grotesque_ family is called `token-amount`, not `token-balance`.

Additional changes:
- Removed `chain-identifier` font style property. Removed uses of the property since it became deprecated.

### CODE REVIEW

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Looking good to me! Let's resolve the conflicts with main branch first, before we merge and I will test it out again.

**Review by kpyszkowski — COMMENTED** [review]
The `main` branch has been merged. Works okay. Please confirm and merge 🔥

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -12,8 +12,13 @@
...
     "@ethersproject/constants": "^5.5.0",
-    "@fontsource/ibm-plex-mono": "^4.5.13",
-    "@fontsource/inter": "^4.5.10",
+    "@fontsource/bricolage-grotesque": "^5.0.4",
+    "@fontsource/dm-sans": "^5.0.18",
+    "@keep-network/coverage-pools": "development",
+    "@keep-network/keep-core": "development",
+    "@keep-network/keep-ecdsa": "development",
+    "@keep-network/random-beacon": "development",
+    "@keep-network/tbtc": "development",
```
[the remark this answers is not in the document]
Comment (kpyszkowski) [reply package.json]: Ref commit: [306b7c3](https://github.com/threshold-network/bitcoin-on-base/pull/30/commits/306b7c3322fa782bb76893a6ff23f1ff38ee61e8)

**Review by michalsmiarowski — APPROVED** [review]
LGTM

**Pull request merged**